hello-buck-cxx/hello-string.cxx

```diff
@@ -1,7 +1,13 @@
+#ifdef  BUCKSTRING
+#define HELLOSTRING "Buck"
+#else
+#define HELLOSTRING "Zuck"
+#endif
+
 #include "stdio.h"
 #include "hello-string.h"
 
-const char * hello_string::HELLO = "Buck";
+const char * hello_string::HELLO = HELLOSTRING;
```

Comment:
I know this is how you found it, but I think I'd rather we just used std::string. This is c++ after all :)